src/styles/block/card.scss

```diff
@@ -0,0 +1,100 @@
+.card__button {
+  display: flex;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  color: rgb(255, 255, 255);
+  align-self: center;
+  align-items: center;
+  cursor: pointer;
+  text-decoration: none;
+  border: 1px solid transparent;
+  background-color: var(--acent-color);
+}
+
+.card__button:hover {
+  border: 1px solid #00acdc;
+  background-color: #fff;
+  color: var(--acent-color);
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
move to `*` selector to prevent duplicating everywhere

Reply:
done